### DISCUSSION TITLE
Look into future

### DISCUSSION THREAD

**Discussion by JetFlightGit** [iSoron/uhabits#2042] [Feature Requests]
In addition to all the things i have mentionied in my other post. This might be nice thing to add which is not too complicated probably and would help a lot. Move the "today" x amount to left so you can see into the future. Could be a simple option but i can see what task are on the horizon to do the next few days. I have added a screenshot where i moved today in middle and gave it a small highlight:

![image](https://github.com/user-attachments/assets/d42ee48d-505e-46ed-a2f6-32dba6d6245a)

**Comment by ezio-melotti** [comment]
This has already been suggested here:
* https://github.com/iSoron/uhabits/discussions/372
* https://github.com/iSoron/uhabits/discussions/1160
* https://github.com/iSoron/uhabits/discussions/1998

**Comment by JetFlightGit** [comment]
Sry my Bad, should have looked through all of posts. But I guess it seems to be something that people like. Who knows if it's too difficult to implement. Seems to be quiet old the Post
